LunrCore/LunrCore.csproj

```diff
@@ -16,6 +16,7 @@
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <LangVersion>9</LangVersion>
     <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Any way we can do this only for NS2? Or better, don't do the polyfills at all and #ifdef the code instead. I'd really really prefer not to allow unsafe.